dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -3038,15 +3038,112 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/Notification'
+              $ref: '#/components/schemas/PublicNotification'
         description: Notification to create
         required: true
       responses:
         default:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Notification'
+                $ref: '#/components/schemas/PublicNotification'
+          description: default response
+      security:
+      - BEARER: []
+      tags:
+      - curation
+  /curation/notifications/user:
+    get:
+      description: Return all notifications for a user
+      operationId: getUserNotifications
+      parameters:
+      - description: "Start index of paging. Pagination results can be based on numbers\
+          \ or other values chosen by the registry implementor (for example, SHA values).\
+          \ If this exceeds the current result set return an empty set.  If not specified\
+          \ in the request, this will start at the beginning of the results."
+        in: query
```

Comment:
Was probably copied, wondering whether we'll truly have so many notifications that they need to be paged :laughing: